pkgs/applications/blockchains/alfis/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config
+, withGui ? false, webkitgtk, Cocoa ? null, WebKit ? null
```

```suggestion
, withGui ? false, webkitgtk, Cocoa, WebKit
```
Those can never be null.